tests/test_callprofiler.py

```diff
@@ -285,6 +298,12 @@ def baz():
         self.assertGreaterEqual(baz_dict["cumulative_time"], time3)
         self.assertGreaterEqual(baz_dict["cumulative_per_call"], time3)
 
+        # Check the ninth line
+        ref_line9 = (
+            "-" * 135
+        )
+        self.assertEqual(stat_line_list[8], ref_line9)
+
```

Comment:
The ninth line is the bottom line of the table. This line is for checking the ninth line of the table.